src/marketingMessage/MarketingMessage.tsx

```diff
@@ -1,11 +1,15 @@
 import React from 'react'
 import cc from 'classcat'
 
+import { StyledMarketingMessage } from './MarketingMessage.style'
+
 export type MarketingMessageProps = Readonly<{
   children: React.ReactNode
   className?: string
 }>
 
 export const MarketingMessage = ({ children, className }: MarketingMessageProps) => (
-  <div className={cc(className)}>{children}</div>
+  <StyledMarketingMessage>
```

Comment:
You'll have the classname on the wrong element here (no on the wrapper anymore). You should do:

```
<StyledMarketingMessage  className={cc(className)}>{children}</StyledMarketingMessage>
```